CMakeLists.txt

```
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.23)
```

```suggestion
cmake_minimum_required(VERSION 3.12)
```
I'm too lazy to update cmake and it works the same in 3.12